allure-app-under-test/src/test/java/com/capgemini/ntc/security/access/DirectoryBrowsingTest.java

```diff
@@ -0,0 +1,71 @@
+package com.capgemini.ntc.security.access;
+
+import static io.restassured.RestAssured.given;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import com.capgemini.ntc.security.EnvironmentParam;
+import com.capgemini.ntc.security.SecurityPage;
+import com.capgemini.ntc.security.SubUrlEnum;
+import com.capgemini.ntc.security.session.SessionEnum;
+
+import io.restassured.specification.RequestSpecification;
+
+/**
+ * The test verifies that directory browsing is disabled.
+ * OWASP ASVS requirement V4.5: Verify that directory browsing is disabled
+ * unless deliberately desired. Additionally, applications should not allow
+ * discovery or disclosure of file or directory metadata, such as Thumbs.db,
+ * .DS_Store, .git or .svn folders.
+ * Purpose: You don't want the attacker to learn about the system more then
+ * needed and he certainly does not need to list files in the directories
+ * on your server (unless you deliberately desire this feature). Putting
+ * private content of your e.g. .git folder into your distribution package
+ * to be uploaded to the server and downloaded by the attacker because of
+ * directory browsing feature is a mortal sin on its own.
+ *
+ * @author Marek Puchalski, Capgemini
+ */
+@RunWith(Parameterized.class)
+public class DirectoryBrowsingTest extends SecurityPage {
```

Comment:
Unfortunately  every "Test" file must extends from "BaseTest"  only then whole mechanism of Observers, Data Driven , Parametrized, Parallel, and so on,  are enabled. 

Could you please modify this.